src/vnode.ts

```diff
@@ -1,49 +0,0 @@
-import {Hooks} from './hooks';
-import {AttachData} from './helpers/attachto'
-import {VNodeStyle} from './modules/style'
-import {On} from './modules/eventlisteners'
-import {Attrs} from './modules/attributes'
-import {Classes} from './modules/class'
-import {Props} from './modules/props'
-import {Dataset} from './modules/dataset'
-import {Hero} from './modules/hero'
-
-export type Key = string | number;
-
-export interface VNode {
-  sel: string | undefined;
-  data: VNodeData | undefined;
-  children: Array<VNode | string> | undefined;
-  elm: Node | undefined;
-  text: string | undefined;
-  key: Key | undefined;
-}
-
-export interface VNodeData {
-  props?: Props;
-  attrs?: Attrs;
-  class?: Classes;
-  style?: VNodeStyle;
-  dataset?: Dataset;
-  on?: On;
-  hero?: Hero;
-  attachData?: AttachData;
-  hook?: Hooks;
-  key?: Key;
-  ns?: string; // for SVGs
-  fn?: () => VNode; // for thunks
-  args?: Array<any>; // for thunks
-  [key: string]: any; // for any other 3rd party module
-}
-
-export function vnode(sel: string | undefined,
```

Comment:
this functionality is dropped entirely, instead, you either use `h`, or `toVNode` or you build your own vnode object manually, not need for such simple abstraction.